io.sloeber.core/build.properties

```diff
@@ -9,7 +9,8 @@ bin.includes = .,\
                config/,\
                OSGI-INF/,\
                src/io/sloeber/core/templates/sketch.ino,\
-               lib/
+               lib/,\
+               lib/jmdns-3.5.3.jar
```

Comment:
As there are multiple jars there I think it is better to have them all mentioned or none. Now there is just one.
PS only saying this as you asked my review :-)

Reply:
Yes thanks. That build.properties is a mess,

Reply:
I don't feel qualified to confirm this.
I'm also thinking: The mess has probably some correlation with my sentence just before this one. ;-)